bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -116,4 +116,14 @@ impl WasmBezier {
 		let line: [DVec2; 2] = js_points.into_serde().unwrap();
 		self.0.line_intersection(line).iter().map(|&p| vec_to_point(&p)).collect::<Vec<JsValue>>()
 	}
+
+	pub fn hull(&self, t: f64) -> JsValue {
+		let hull = self
+			.0
+			.hull(t)
+			.iter()
+			.map(|level| level.iter().map(|&point| Point { x: point.x, y: point.y }).collect::<Vec<Point>>())
+			.collect::<Vec<Vec<Point>>>();
+		JsValue::from_serde(&serde_json::to_string(&hull).unwrap()).unwrap()
```

Comment:
Rebase this branch off master, and then use `to_js_value(hull)` here. `to_js_value` was added in #711

Reply:
Added. Thanks!